pkgs/by-name/st/stem/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchurl
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "stem";
+  version = "1.8.2";
+
+  src = fetchurl {
+    url = "https://gitlab.torproject.org/tpo/network-health/stem/-/archive/master/stem-master.zip";
+    sha256 = "sha256-qHvg/T8oU9vtyPWK/fXxIPJlg1FPCc76SzdOTw5iNcQ=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools ];
+
+  dontUseSetuptoolsCheck = true;
+
+  meta = with lib; {
+    description = "Stem is a Python controller library for Tor.";
```

Comment:
description must not end with a period
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes